query/src/main/java/org/infinispan/query/backend/QueryInterceptor.java

```diff
@@ -138,19 +135,18 @@ public void prepareForStopping() {
       stopping.set(true);
    }
 
+   //todo [anistor] parameter 'value' is never used
```

Comment:
Can we remove unused parameters from here @gustavonalle ?

Reply:
I'll remove that then and push an update now to have a new CI run while I work on the docs aspect of it.